providers/fetch/mavenBasedFetch.js

```diff
@@ -23,13 +23,11 @@ const extensionMap = {
   jar: '.jar'
 }
 
-const defaultHeaders = { headers: { 'User-Agent': 'clearlydefined.io crawler (clearlydefined@outlook.com)' } }
-
 class MavenBasedFetch extends AbstractFetch {
   constructor(providerMap, options) {
     super(options)
     this._providerMap = { ...providerMap }
-    this._handleRequestPromise = options.requestPromise || withDefaults(defaultHeaders)
+    this._handleRequestPromise = options.requestPromise || callFetch
     this._handleRequestStream = options.requestStream || nodeRequest.defaults(defaultHeaders).get
```

Comment:
This line should be rewritten as 
this._handleRequestStream = options.requestStream || nodeRequest.defaults({ headers: defaultHeaders }).get in order to maintain the previous arrangement. Please let me know if this sounds good to you.